<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>801</width>
        <height>571</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>60</number>
       </property>
       <property name="topMargin">
        <number>120</number>
       </property>
       <property name="rightMargin">
        <number>60</number>
       </property>
       <property name="bottomMargin">
        <number>120</number>
       </property>
       <property name="spacing">
        <number>7</number>
       </property>
       <item row="3" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="t1stat_label">
           <property name="text">
            <string>Кратчайший Путь: 0 (0/0)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="t2stat_label">
           <property name="text">
            <string>Метод Деления Пополам: 0 (0/0)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="t3stat_label">
           <property name="text">
            <string>Шифр Виженера: 0 (0/0)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Get statistic</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="task3Button">
           <property name="text">
            <string>Шифр Виженера</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exitButton">
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="task1Button">
           <property name="text">
            <string>Кратчайший Путь</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="task2Button">
           <property name="text">
            <string>Метод Деления Пополам</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>651</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="t1pushButton">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="returnButton1">
           <property name="text">
            <string>Return</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="t1_label">
           <property name="text">
            <string>Для начала работы нажмите на кнопку &quot;Обновить задачу&quot;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t1_lineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Введите ответ сюда</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="t1_getTaskpushButton">
         <property name="text">
          <string>Обновить задачу</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_t1_top">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Кратчайший Путь</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_t2_top">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Метод Деления Пополам</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>651</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="t2pushButton">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="returnButton2">
           <property name="text">
            <string>Return</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="t2_label">
           <property name="text">
            <string>Для начала работы нажмите на кнопку &quot;Обновить задачу&quot;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t2_lineEdit">
           <property name="placeholderText">
            <string>Введите ответ сюда</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="t2_getTaskpushButton">
         <property name="text">
          <string>Обновить задачу</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="alert_t2">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>520</y>
        <width>791</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>491</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ответ к каждому заданию является значение на 5-й итерации метода</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>651</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="t3pushButton">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="returnButton3">
           <property name="text">
            <string>Return</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="t3_label">
           <property name="text">
            <string>Для начала работы нажмите на кнопку &quot;Обновить задачу&quot;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="t3_lineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Введите ответ сюда</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="t3_getTaskpushButton">
         <property name="text">
          <string>Обновить задачу</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_t3_top">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Шифр Виженера</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
